.nvmrc

```diff
@@ -1 +1 @@
-10
+12
```

Comment:
This change was for some of the eslint dependencies; they were crying for node v12. This shouldn't affect what node versions the library supports, as indicated in the `package.json`.